docs/developer/adding-community-components.md

```diff
@@ -0,0 +1,67 @@
+# Adding community components
+
+[Community components][cc] are components that are implemented and maintained by the community.
+
+## Community vs core components
+
+The community components category is mainly targeted at vendor-specific components for which Grafana does not offer commercial support (for example the Datadog exporter).
```

Comment:
Are there other targets? or is it just for vendor-specific components?

Reply:
this is the primary target but we could also accept a vendor-neutral component as community component and not core component if we don't want to maintain it. I don't have an example for it and it's not something that we will do in the near future though (the first candidates for the community components are all vendor-specific)